liquibase-standard/src/main/java/liquibase/sqlgenerator/core/CreateIndexGenerator.java

```diff
@@ -140,6 +157,28 @@ public Sql[] generateSql(CreateIndexStatement statement, Database database, SqlG
 	    }
 	    buffer.append(")");
 
+        //support for include statements.
```

Comment:
Same here.

Reply:
for this one includedcolumns.size() is guaranteed to be zero for non-mysql by line 125 (the only code that populates includedColumns is inside that if on database type).